src/analysis/normed_space/star/multiplier.lean

```diff
@@ -0,0 +1,446 @@
+end
+
+noncomputable instance : normed_ring 𝓜(𝕜, A) :=
+{ norm_mul := λ a b,
+    begin
+      refine max_le ((norm_mul_le _ _).trans _) ((norm_mul_le _ _).trans _),
+      exact mul_le_mul (le_max_left _ _) (le_max_left _ _) (norm_nonneg _)
+        ((norm_nonneg _).trans $ le_max_left _ _),
+      exact mul_comm (‖a.snd‖) (‖b.snd‖) ▸ mul_le_mul (le_max_right _ _) (le_max_right _ _)
+        (norm_nonneg _) ((norm_nonneg _).trans $ le_max_right _ _),
+    end,
+  .. double_centralizer.ring,
+  .. double_centralizer.normed_add_comm_group }
+
+noncomputable instance : normed_algebra 𝕜 𝓜(𝕜, A) :=
+{ ..double_centralizer.algebra, ..double_centralizer.normed_space }
+
+variables [star_ring A] [cstar_ring A]
+
+/-- For `a : 𝓜(𝕜, A)`, the norms of `a.fst` and `a.snd` coincide, and hence these
+also coincide with `‖a‖` which is `max (‖a.fst‖) (‖a.snd‖)`. -/
+lemma norm_fst_eq_snd (a : 𝓜(𝕜, A)) : ‖a.fst‖ = ‖a.snd‖ :=
+begin
+  -- a handy lemma for this proof
+  have h0 : ∀ f : A →L[𝕜] A, ∀ C : ℝ≥0, (∀ b : A, ‖f b‖₊ ^ 2 ≤ C * ‖f b‖₊ * ‖b‖₊) → ‖f‖₊ ≤ C,
+  { intros f C h,
+    have h1 : ∀ b, C * ‖f b‖₊ * ‖b‖₊ ≤ C * ‖f‖₊ * ‖b‖₊ ^ 2,
+    { intros b,
+      convert mul_le_mul_right' (mul_le_mul_left' (f.le_op_nnnorm b) C) (‖b‖₊) using 1,
+      ring, },
+    have := div_le_of_le_mul (f.op_nnnorm_le_bound _ (by simpa only [sqrt_sq, sqrt_mul]
+      using (λ b, sqrt_le_sqrt_iff.mpr ((h b).trans (h1 b))))),
+    convert rpow_le_rpow this two_pos.le,
+    { simp only [rpow_two, div_pow, sq_sqrt], simp only [sq, mul_self_div_self] },
+    { simp only [rpow_two, sq_sqrt] } },
+  have h1 : ∀ b, ‖a.fst b‖₊ ^ 2 ≤ ‖a.snd‖₊ * ‖a.fst b‖₊ * ‖b‖₊,
+  { intros b,
+    calc ‖a.fst b‖₊ ^ 2
+        = ‖star (a.fst b) * (a.fst b)‖₊
+        : by simpa only [←sq] using (cstar_ring.nnnorm_star_mul_self).symm
+    ... ≤ ‖a.snd (star (a.fst b))‖₊ * ‖b‖₊ : a.central (star (a.fst b)) b ▸ nnnorm_mul_le _ _
+    ... ≤ ‖a.snd‖₊ * ‖a.fst b‖₊ * ‖b‖₊
+        : nnnorm_star (a.fst b) ▸ mul_le_mul_right' (a.snd.le_op_nnnorm _) _},
+  have h2 : ∀ b, ‖a.snd b‖₊ ^ 2 ≤ ‖a.fst‖₊ * ‖a.snd b‖₊ * ‖b‖₊,
+  { intros b,
+    calc ‖a.snd b‖₊ ^ 2
+        = ‖a.snd b * star (a.snd b)‖₊
+        : by simpa only [←sq] using (cstar_ring.nnnorm_self_mul_star).symm
+    ... ≤ ‖b‖₊ * ‖a.fst (star (a.snd b))‖₊
+        : (a.central b (star (a.snd b))).symm ▸ nnnorm_mul_le _ _
+    ... = ‖a.fst (star (a.snd b))‖₊ * ‖b‖₊ : mul_comm _ _
+    ... ≤ ‖a.fst‖₊ * ‖a.snd b‖₊ * ‖b‖₊
+        : nnnorm_star (a.snd b) ▸ mul_le_mul_right' (a.fst.le_op_nnnorm _) _  },
+  exact le_antisymm (h0 _ _ h1) (h0 _ _ h2),
+end
+
+lemma norm_fst (a : 𝓜(𝕜, A)) : ‖a‖ = ‖a.fst‖ :=
+  by simp only [norm_eq, norm_fst_eq_snd, max_eq_right, eq_self_iff_true]
+lemma norm_snd (a : 𝓜(𝕜, A)) : ‖a‖ = ‖a.snd‖ := by rw [norm_fst, norm_fst_eq_snd]
+lemma nnnorm_fst (a : 𝓜(𝕜, A)) : ‖a‖₊ = ‖a.fst‖₊ := subtype.ext (norm_fst a)
+lemma nnnorm_snd (a : 𝓜(𝕜, A)) : ‖a‖₊ = ‖a.snd‖₊ := subtype.ext (norm_snd a)
```

Comment:
Can you turn those around and `@[simp]` them?

Reply:
Apparently I can't add `simp` to the first two because the `simp_nf` linter times out. I think maybe it's because `∥a∥ := max ∥a.fst∥ ∥a.snd∥` and so it gets stuck in a loop? But I have reversed the direction of all of these and added `simp` to the latter two.